src/test/prediction_service_test.cpp

```diff
@@ -194,6 +233,31 @@ class TestPredict : public ::testing::Test {
         ASSERT_EQ(0, std::memcmp(actualValues.data(), expectedValues.data(), expectedValues.size() * sizeof(float)))
             << readableError(expectedValues.data(), actualValues.data(), expectedValues.size() * sizeof(float));
     }
+    static void checkOutputValues(const ovms::InferenceResponse& res, const std::vector<float>& expectedValues, const std::string& outputName = INCREMENT_1x3x4x5_MODEL_OUTPUT_NAME) {
+        InferenceResponse& response = const_cast<InferenceResponse&>(res);  // TODO decide if output should be const
```

Comment:
Couldnt response be const if we make getOutput const as well? To make getOutput const we would need to return const InferenceTensor reference, right?

Reply:
Yes and also return const buffer in 
OVMS_InferenceResponseGetOutput(..., const void** data)
instead of
OVMS_InferenceResponseGetOutput(..., void** data)
Here & for now it does not make any harm and I am not sure that we should block user from receiving non-const buffer

Reply:
Decided to change to OVMS_InferenceResponseGetOutput(..., const void** data)